Prox/Prox/Utilities/EventNotificationsManager.swift

```diff
@@ -6,8 +6,20 @@ import Foundation
 import CoreLocation
 import UserNotifications
 
+private let sentNotificationDictKey = "sent_notifications_dict"
+
 class EventNotificationsManager {
 
+    fileprivate var sentNotifications: [String: [String]] {
+        get {
+            return UserDefaults.standard.dictionary(forKey: sentNotificationDictKey) as? [String : [String]] ?? [String : [String]]()
```

Comment:
Good point. I'm sure we can use the start time of the event at a place to resolve this, and clear out anything from the cache that is not today. I'll also use an NSCache and only store to `UserDefaults` when we go into the background, rather than storing in `UserDefaults` by default